portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -63,6 +122,43 @@ definition {
 			value1 = "${emptyMessage}");
 	}
 
+	macro clearAllFilter {
+		Click(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_FILTER_CLEAR_ALL");
+	}
+
+	macro create6UsersAndAssign {
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		for (var count : list "1,2,3,4,5,6") {
+			BlogsNavigator.gotoAddEntry();
+
+			BlogsEntry.addEntryContent(
+				entryContent = "test content ${count}",
+				entryTitle = "Test ${count}");
```

Comment:
Hi @calvinctvo ,
Please use JSONBlog.addEntry macro to add Blog entries. It is faster :)